chainio/log.go

```diff
@@ -0,0 +1,29 @@
+package chainio
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var clog btclog.Logger
+
+// The default amount of logging is none.
+func init() {
+	UseLogger(build.NewSubLogger("CHIO", nil))
```

Comment:
Should add this as a `Subsystem` variable in the package, so it can be used in the root `log.go` file without needing to re-use the value.

Reply:
fixed